### PR TITLE
feat: label when the build fails

### PR DESCRIPTION
**Pull request by Panquesito7**
#### Description of Change

- Label `Autochecks are failing` when the build fails.

#### References

None.

#### Checklist

- [x] Added description of change
- [ ] ~Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)~
- [ ] ~Added tests and example, test must pass~
- [ ] ~Relevant documentation/comments is changed or added~
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: .github/workflows/awesome_workflow.yml
Comment (tjgurwara99) [inline .github/workflows/awesome_workflow.yml:110]: Same comment as the comment in the PR in C++ repo

> Should we not have a scheduled job for checking for failures on all of the CI's instead of just checking for `failure` on this Action?

Comment (Panquesito7) [reply .github/workflows/awesome_workflow.yml:110]: https://github.com/TheAlgorithms/C-Plus-Plus/pull/2455#discussion_r1183037035

**Review by tjgurwara99 — APPROVED** [review]
Oh sorry, forgot about this PR! LGTM

**Review by alexpantyukhin — DISMISSED** [review]

**Review by tjgurwara99 — APPROVED** [review]
Another reason I don't like direct merging of PR without explicit approval from maintainers 😭 It gets rid of the approvals when we do update our branch to sync with main/master

File: .github/workflows/awesome_workflow.yml
Comment (Panquesito7) [reply .github/workflows/awesome_workflow.yml:110]: Actually, we should see if there's a possible way to do this. I'll be merging this for now, though.

**Pull request merged**
